src/components/QnA/comps/ModalTemplate.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import styled from 'styled-components';
+import FlexColumn from '../styles/FlexColumn.styled';
+
+const OuterFlexColumn = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+`;
+
+const FlexForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  max-width: none;
+`;
+
+const ModalTitle = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const ModalSubtitle = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const Label = styled.label`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const FirstInput = styled.textarea.attrs({ maxLength: '1000', required: 'required' })`
+display: block;
+padding: 0 0 15px 0;
+width: 100%;
+overflow: auto;
+height: 65px;
+line-height: 1.2;
+`;
+
+const SecondInput = styled.input.attrs({ maxLength: '60', required: 'required' })`
+padding: 0 0 15px 0;
+`;
+
+const ThirdInput = styled.input.attrs({ maxLength: '60', required: 'required', type: 'email' })`
+  padding: 0 0 15px 0;
+`;
+
+const SubmitModal = styled.button`
+
+`;
+
+const NicknameParagraph = styled.p`
+  opacity: 50%;
+  color: black;
+  position: absolute;
```

Comment:
`NicknameParagraph` and `EmailParagraph` are pretty similar, the only difference being their positioning. Is it possible to refactor this to either use one component in both places, or just have this component take in props to set the `bottom` positioning? It might even be possible to combine them with `ThousandCharsParagraph` and turn all three into one component. Also, the paragraph gets put on a new line, but I think it would look a bit better if it was on the same line as the input. Let me know what you think about that, since it's up to you.

Reply:
Yeah, on of the problems I was having with textarea instead of input is that since it puts it on a new line everytime, it overflows and goes through the characters left paragraph. I am looking into solutions, but I do not have any yet.